exercises/5.1/solution_ws/ros2/src/lesson_perception/src/pcd_to_pointcloud.cpp

```diff
@@ -0,0 +1,55 @@
+#include <chrono> // std::chrono::nanoseconds
+#include <memory> // std::make_shared()
+#include <string> // std::string
+
+#include <pcl/io/pcd_io.h>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+#include <pcl_conversions/pcl_conversions.h>
+#include <rclcpp/rclcpp.hpp>
+#include <sensor_msgs/msg/point_cloud2.hpp>
+
+int main(int argc, char **argv)
```

Comment:
I'm not sure if we're consistent on this throughout the training, but I believe the ROS Style guide / Google Style Guide recommends putting `char** argv` not `char **argv` - the double-pointer is part of the type, not the name.